src/planscape/conditions/raster_utils_test.py

```diff
@@ -64,42 +192,42 @@ def test_computes_mean_scores(self):
     def test_raises_error_for_missing_geo(self):
         plan = Plan.objects.create(geometry=None, region_name=self.region)
 
-        foo_raster = self._create_raster(4, 4, (1, 2, 3, 4,
-                                                5, 6, 7, 8,
-                                                9, 10, 11, 12,
-                                                13, 14, 15, 16))
-        self._create_condition_db("foo", "foo_normalized", foo_raster)
+        foo_raster = _create_raster(self, 4, 4, (1, 2, 3, 4,
+                                                 5, 6, 7, 8,
+                                                 9, 10, 11, 12,
+                                                 13, 14, 15, 16))
+        _create_condition_db(self, "foo", "foo_normalized", foo_raster)
         with self.assertRaises(Exception) as context:
             fetch_or_compute_mean_condition_scores(plan)
         self.assertEqual(
             str(context.exception), "plan is missing geometry")
 
     def test_raises_error_for_bad_region(self):
-        geo = self._create_geo(0, 3, 0, 1)
+        geo = _create_geo(self, 0, 3, 0, 1)
         plan = Plan.objects.create(
             geometry=geo, region_name="nonsensical region")
 
-        foo_raster = self._create_raster(4, 4, (1, 2, 3, 4,
-                                                5, 6, 7, 8,
-                                                9, 10, 11, 12,
-                                                13, 14, 15, 16))
-        self._create_condition_db("foo", "foo_normalized", foo_raster)
+        foo_raster = _create_raster(self, 4, 4, (1, 2, 3, 4,
+                                                 5, 6, 7, 8,
+                                                 9, 10, 11, 12,
+                                                 13, 14, 15, 16))
+        _create_condition_db(self, "foo", "foo_normalized", foo_raster)
         with self.assertRaises(Exception) as context:
             fetch_or_compute_mean_condition_scores(plan)
         self.assertEqual(
             str(context.exception),
             "no conditions exist for region, nonsensical region")
```

Comment:
i'm not sure if this is the error we should return. we should check that the region is valid prior to querying for any conditions. wdyt?

Reply:
is there a library for checking region validity?

Reply:
Done.